benchmarks/scripts/common.py

```diff
@@ -106,9 +108,111 @@ def qiskit_compile(qiskit_circuit):
     )
 
 
+class BenchmarkTargetGateset(cirq.TwoQubitCompilationTargetGateset):
+    """Target gateset for compiling circuits for benchmarking.
+
+    This is based on the IonQTargetGateset in cirq_ionq package
+
+    The gate families accepted by this gateset are:
+
+    Instance gate families:
+    *  Single-Qubit Gates: `cirq.H`, `cirq.Rx`, `cirq.Ry`, `cirq.Rz`.
+    *  Two-Qubit Gates: `cirq.CNOT`
+    *  `cirq.MeasurementGate`
+    """
+
+    def __init__(self, *, atol: float = 1e-8):
+        """Initializes BenchmarkTargetGateset
+
+        Args:
+            atol: A limit on the amount of absolute error introduced by the decomposition.
+        """
+        super().__init__(
+            cirq.H,
+            cirq.CNOT,
+            cirq.Rx,
+            cirq.Ry,
+            cirq.Rz,
+            cirq.MeasurementGate,
+            unroll_circuit_op=False,
+        )
```

Comment:
Another possibility is setting `preserve_moment_structure` to `False` . In that case, logical equivalence would need to be checked in addition to target gateset, but I think that's fine.

Reply:
So I understand, that would enable these passes to reorder more of the circuit?

Do any of our benchmark circuits have moments/ordering specified?

Reply:
Yes, I expect it would allow reordering of the circuit. As far as I know, none of our benchmarking circuits have ordering specified.